reco_utils/recommender/deeprec/deeprec_utils.py

```diff
@@ -389,6 +394,14 @@ def cal_metric(labels, preds, metrics):
             pred[pred < 0.5] = 0
             f1 = f1_score(np.asarray(labels), pred)
             res["f1"] = round(f1, 4)
+        elif metric == 'mae':
```

Comment:
seems to be some inconsistency with rounding to 4 decimal places, is that something that should happen here too? also, noticing that the rmse is rounding prior to sqrt instead of after